doc/getting_and_using_mir.md

```diff
@@ -31,6 +30,16 @@ You can install the Mir examples along with the Mir graphics drivers as follows:
     $ sudo apt install mir-demos qterminal
     $ sudo apt install mir-graphics-drivers-desktop qtubuntu-desktop
 
+## Getting Mir examples on Fedora
+
+On Fedora Mir is available from the archive. We recommend using qterminal with
+Mir as the Qt Wayland backend currently works better with Mir's Wayland support
+than other options.   
```

Comment:
I find this sentence confusing. What would be other options than "Qt Wayland" for "Mir's Wayland"?